shoplist/src/main/res/layout/main_fragment.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Use code style like:
Ids should start with special characters:
for TextView use something like this:
android:id="@+id/tvTitle" instead of android:id="@+id/title"
for AppCompatImageView:
android:id="@+id/ivLatestListIcon" instead of android:id="@+id/latest_list_icon"
The same for ProgressBar and RecyclerView (add id so you can use it later)
add proper naming for ids

Views are not constrained normally (TextView, AppCompatImageView, RecyclerView)
Don't hard code strings (android:text="Check out your shopping lists!")
Don't hard code dimens (android:padding="8dp"), move to resource folder